recipes/notcurses/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, CMake, tools
+
+
+class NotcursesConan(ConanFile):
+    name = "notcurses"
+    description = "a blingful TUI/character graphics library"
+    topics = ("graphics", "curses", "tui", "console", "ncurses")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://nick-black.com/dankwiki/index.php/Notcurses"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "openimageio/2.2.7.0", "libunistring/0.9.10"
+    options = {"shared": [True, False]}
+    default_options = {"shared": True,
+                       "with_doctest": True}
+    generators = "cmake"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder="notcurses")
+        cmake.build()
+
+        # Explicit way:
+        # self.run('cmake %s/notcurses %s'
+        #          % (self.source_folder, cmake.command_line))
+        # self.run("cmake --build . %s" % cmake.build_config)
+
+    def package(self):
+        self.copy("*.h", dst="include", src="notcurses")
```

Comment:
```suggestion
        self.copy("*.h", dst="include", src=self._source_subfolder)
```

Reply:
But you should rely on CMake installation in `package()`

Reply:
done, thanks!